Gem/Code/Source/AutoGen/NetworkTeleportCompatibleComponent.AutoComponent.xml

```diff
@@ -11,14 +11,29 @@
 	<ComponentRelation Constraint="Required" HasController="true" Name="NetworkTransformComponent" 
                        Namespace="Multiplayer" Include="Multiplayer/Components/NetworkTransformComponent.h" />
 
+    <Include File="Source/Effects/GameEffect.h" />
+
+    <ArchetypeProperty Type="GameEffect"  Name="TeleportEffect"  Init="" Container="Object" Count="1" ExposeToEditor="true" Description="Specifies the effect to play upon teleportation" />
```

Comment:
Could remove "Count". "Count" attribute isn't required if Container = Object (only array/vectors need it). It's one of the few attrib that we dynamically don't try to read :D